unity/Assets/Scripts/PhysicsRemoteFPSAgentController.cs

```diff
@@ -5829,125 +5727,75 @@ protected IEnumerator ToggleAndWait(CanToggleOnOff ctof)
             actionFinished(success);
         }
 
-        public void OpenObject(ServerAction action) {
-            SimObjPhysics target = null;
-            if (action.forceAction) {
-                action.forceVisible = true;
-            }
-            //no target object specified, so instead try and use x/y screen coordinates
-            if(action.objectId == null)
-            {
-                if(!ScreenToWorldTarget(action.x, action.y, ref target, !action.forceAction))
-                {
-                    //error message is set insice ScreenToWorldTarget
-                    actionFinished(false);
-                    return;
-                }
+        // private helper used with OpenObject commands
+        private void openObject(
```

Comment:
I don't like that this helper function controls whether or not the action is finished, what if I want to call this as part of some other method in which I want to do something after having opened the object?

Reply:
Added an option with `markActionFinished`.